db/version_set.cc

```diff
@@ -5681,6 +5691,25 @@ uint64_t VersionSet::GetTotalSstFilesSize(Version* dummy_versions) {
   return total_files_size;
 }
 
+uint64_t VersionSet::GetTotalBlobFileSize(Version* dummy_versions) {
+  std::unordered_set<uint64_t> unique_blob_files;
+  uint64_t total_blob_file_size = 0;
```

Comment:
One more thing: let's rename this to something else (e.g. `blob_file_size_sum` ?) to fix the amalgamated ("unity") build.